hal/network/ncp/cellular/cellular_network_manager.h

```diff
@@ -45,12 +46,17 @@ class CellularNetworkConfig {
     const char* password() const;
     bool hasPassword() const;
 
+    CellularNetworkConfig& netProv(CellularNetworkProv net_prov);
```

Comment:
Snake case was used here, and the private variables are already snaking with abc_ so it seeeemed appropriate :D If you don't mind too much, I'd like to leave it snake_case since it's internal.
```
struct NetworkConfig {
    CellularNetworkProv net_prov;
    const char* mcc;
    const char* mnc;
    const char *iccid_prefix;
    const char* apn;
    const char* user;
    const char* password;
};
```

Reply:
No strong preferences, but these are two different things though. I'm not commenting on `NetworkConfig`, but the rest of `CellularNetworkConfig` uses camelCase.

Reply:
Will change these to something else to be unique and avoid case issues.